addons/hr_expense/hr_expense.py

```diff
@@ -112,6 +112,26 @@ def _get_currency(self, cr, uid, context=None):
         'currency_id': _get_currency,
     }
 
+    def add_follower(self, cr, uid, ids,employee_id, context=None):
+        employee = self.pool.get('hr.employee').browse(cr, uid, employee_id, context=context)
+        if  employee and employee.user_id:
+            self.message_subscribe_users(cr, uid, ids, user_ids=[employee.user_id.id], context=context)
+        return True
+
+    def create(self, cr, uid, values, context=None):
+        if context is None:
```

Comment:
use `if not context`, else you will create a new `{}` in the case where context is already `{}`